### DISCUSSION TITLE
Add a button to Quickly toggle split screen tabs

### DISCUSSION THREAD

**Discussion by Adish08** [zen-browser/desktop#3771] [Ideas]
I like that in MS Edge, when I require a split tab next to my current tab, I simply click on a button, and I am good to go, it opens up the search page (and an ability to modify the address of the default page with custom URL would be great) where I can open anything.

Currently in Zen, I have to open a New Tab first and then select the tabs and split it, which I personally found to be very tedious.

Here is a screen recording of my suggestion:

https://github.com/user-attachments/assets/902d40b0-2cf0-4664-a56d-9de24ad1ac12

**Comment by Michael679089** [comment]
![image](https://github.com/user-attachments/assets/bcf022f5-5264-493d-823a-5cfa0f064ef2)
The split tab button can be added to the toolbar optionally. This can be done by the customizing the toolbar option.

**Reply by Propheticus** [reply]
You're showing Edge, Zen does not have such a button. That is the point of this idea.

**Comment by Propheticus** [comment]
Did you know you can drag another tab onto the viewport of the tab you're currently viewing to split them?
It's a bit faster than selecting 2 tabs first and then right-clicking to select 'split 2 tabs'.
Of course it could be a little faster still, like you explained.
